### PR TITLE
Feat(#55) manage session

### PR DESCRIPTION
**Pull request by MelnychenkoIvan**
## PR Checklist

PR meets the following requirements:

- [ ] Changes follow our [guidelines](https://github.com/extreme-startup/questionator/wiki/Contributing)
- [ ] Tests for the changes have been added
- [ ] Docs have been added / updated

## PR Type

PR introduces the following changes:

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## Actual/New behavior

- add new class implementing manage session entity for DB; 

Issue Number: #55

### CODE REVIEW

File: server/src/migration/1555064588740-ManageSessionEntity.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class ManageSessionEntity1555064588740 implements MigrationInterface {
+
```
Comment (kovnick) [inline server/src/migration/1555064588740-ManageSessionEntity.ts]: Please, check this migration on dev server before merge your branch to master. You could notify me for manual deploy.

File: server/src/modules/manage-session/manage-session.controller.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import { ManageSessionService } from './manage-session.service';
+import { ManageSessionDto } from './ManageSession.dto';
+
+@Controller('manage-session')
+export class ManageSessionController {
+  constructor(private msService: ManageSessionService) {}
+
+  @Get()
+  showAllSessions(): Promise<ManageSessionDto[]> {
```
Comment (kovnick) [inline server/src/modules/manage-session/manage-session.controller.ts]: Check that swagger doc generates for these endpoints

Comment (MelnychenkoIvan) [reply server/src/modules/manage-session/manage-session.controller.ts]: I've added swagger docs to entities

File: server/src/migration/1555064588740-ManageSessionEntity.ts (hunk 1 above)
Comment (MelnychenkoIvan) [reply server/src/migration/1555064588740-ManageSessionEntity.ts]: this migration was generated automatically and i checked that, it worked

Comment (kovnick) [reply server/src/migration/1555064588740-ManageSessionEntity.ts]: I understand, but your local env could be different with DBs on dev/qa env. Anyway, I would recommend you to check this migration before merge.

Comment (MelnychenkoIvan) [reply server/src/migration/1555064588740-ManageSessionEntity.ts]: ok, i'll do, but i don't know how i can do that. can you help?

Comment (kovnick) [reply server/src/migration/1555064588740-ManageSessionEntity.ts]: Yep, is this PR finished or still in progress?

Comment (MelnychenkoIvan) [reply server/src/migration/1555064588740-ManageSessionEntity.ts]: finished now

Comment (MelnychenkoIvan) [reply server/src/migration/1555064588740-ManageSessionEntity.ts]: @kovnick need your help!

**Pull request merged**